lib/wpc_proto/internal_modules/proto_config.c

```diff
@@ -1058,7 +1058,7 @@ app_proto_res_e Proto_config_handle_set_scratchpad_target_and_action_request(
                                               param);
 
             // Read otap variable back to be sure everything is updated
-            initialize_otap_variables();
+            Proto_config_refresh_otap_infos();
```

Comment:
Shouldn't it be moved in the else of the if (res != APP_RES_OK) ?

Value is changed only if APP_RES8OK is returned. ANd it would allow to call Proto_config_refresh_otap_infos only one time